projects/adrv9009zu11eg/adrv2crr_fmcomms8/system_project.tcl

```diff
@@ -0,0 +1,27 @@
+
+source ../../scripts/adi_env.tcl
+source $ad_hdl_dir/projects/scripts/adi_project_xilinx.tcl
+source $ad_hdl_dir/projects/scripts/adi_board.tcl
+
+set p_device "xczu11eg-ffvf1517-2-i"
+set sys_zynq 2
+
+adi_project fmcomms8_adrv9009zu11eg 0 [list \
+  JESD_RX_M 16 \
```

Comment:
We are going to support configurations where Samples per frame (S) is always 1  ?

Reply:
Currently only L and M parameters are used as variables. Will add also S. We can improve all projects to better support compile configuration through only changing the parameters, but should be a different effort which applies to all projects. For now, reconfiguration requires changes in the constraints and system_top.v, on top of the system_project parameters

Reply:
Added